components/html-block/html-block.js

```diff
@@ -22,32 +25,32 @@ export const htmlBlockContentStyles = css`
 	}
 	h1 {
 		font-size: 2em;
-		line-height: 37px;
+		line-height: 0.97;
 		margin: 21.43px 0;
 	}
 	h2 {
 		font-size: 1.5em;
-		line-height: 27px;
+		line-height: 0.95;
 		margin: 19.92px 0;
 	}
 	h3 {
 		font-size: 1.2em;
-		line-height: 23px;
+		line-height: 1;
 		margin: 18.72px 0;
 	}
 	h4 {
 		font-size: 1em;
-		line-height: 20px;
+		line-height: 1.05;
```

Comment:
At `20px` font-size, `0.05` translates to `1px` - maybe worthwhile.  The extra `0.01` for `h5` doesn't appear to accomplish much.  Just contemplating whether we need/want to use such values.  🤷

Reply:
Yeah, that's likely true. This one is definitely an outlier in how far away it is from 1 (comparatively). The others are so close that they do really come down to decimal points for any "reasonable" font size.

Reply:
Yeah agreed... seems like we could round these all down to `1` and nobody would notice.